courses/machine_learning/deepdive2/recommendation_systems/solutions/1_als_bqml.ipynb

```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
```

Comment:
<p>Can we add a link so students can explore? :)</p><p>[grouplens.org](<a href="https://grouplens.org/" target="_blank">https://grouplens.org/</a>)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/642/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>